galaxy.yml

```diff
@@ -87,6 +87,7 @@ dependencies:
   'git+https://github.com/ansible-collections/mellanox.onyx': 'master'
   'git+https://github.com/openshift/community.okd': 'main'
   'git+https://github.com/ovirt/ovirt-ansible-collection': 'master'
+  'fedora.linux_system_roles': '1.87.1'
```

Comment:
cant we use github to avoid ansible galaxy usual hiccups? That one is used whenever we run `ansible-galaxy collection install . -f` or related.

Reply:
we can't, i've explained why in a comment above. There's no official repo for the collection.